### DISCUSSION TITLE
How to not register the touch event that wakes up the screen ?

### DISCUSSION THREAD

**Discussion by blemasle** [KlipperScreen/KlipperScreen#242] [Q&A] [answered]
Hi,

Is there a way to make the touch event that brings the screen back from sleep inactive ?
When the screen is blank due to inactivity, you have to touch it to turn it on. But that touch is actually registered as a proper touch event.

So to wake up the screen safely, you have to know the panel you're on and try to roughly aim for a zone you know has no near input. Kinda tricky 😁 

Regards

**Comment by Ka8oum** [comment]
I have the same issue as you, does anybody know how to fix this? I am running Fluidd on a raspberry pi 4 with the official Raspberry 7" touch screen

**Reply by Fr33man85** [reply]
same here

**Comment by alfrix** [comment, marked as answer]
#340
